source/js/options.js

```diff
@@ -129,20 +129,27 @@ $(document).ready(function() {
   showReviewsLink();
   load_options();
 
-  $.ajax("ChangeLog").done(function(text) {
-    $("#changelogText").text(text);
-  });
+  var changeLogRequest = new XMLHttpRequest();
+  changeLogRequest.onreadystatechange = function() {
+    if (changeLogRequest.responseText) {
+      $("#changelogText").text(changeLogRequest.responseText);
+    }
+  }
+  changeLogRequest.open('GET', 'ChangeLog', true);
+  changeLogRequest.send();
```

Comment:
Why is this change needed?

Also, if we're to replace `$.ajax` anywhere, let's be modern and use [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) or at the very least `load` event instead of `onreadystatechange`.